plugin/config_test.go

```diff
@@ -133,3 +133,59 @@ pipeline:
 	require.Error(t, err)
 	require.Contains(t, err.Error(), "reached max plugin depth")
 }
+
+func TestPluginIDs(t *testing.T) {
```

Comment:
this had to be in the plugin package because it created an import cycle in the pipeline package. Above this, there is a buildoperators call.